airbyte-cdk/bulk/build.gradle

```diff
@@ -232,3 +232,5 @@ class BumpBulkCdkVersionTask extends DefaultTask {
 tasks.register('bumpVersion', BumpBulkCdkVersionTask) {
     it.versionFile = versionFile
 }
+
+// force ci run
```

Comment:
👀

Reply:
this is what I get for not switching back to draft mode >.>